lab_1_classify_by_unigrams/main.py

```diff
@@ -3,14 +3,23 @@
 Language detection
 """
 
-
 def tokenize(text: str) -> list[str] | None:
     """
     Splits a text into tokens, converts the tokens into lowercase,
     removes punctuation, digits and other symbols
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    else:
```

Comment:
This `else` is redundant. Can you explain why?

Reply:
Here and everywhere else, actually.

Reply:
else is not needed. the code will work without it. I haven't realized it before...